Core/Core/Files/FileDetails/FileDetailsViewController.swift

```diff
@@ -357,6 +370,23 @@ extension FileDetailsViewController: PDFViewControllerDelegate {
         }
     }
 
+    public func pdfViewController(_ pdfController: PDFViewController, didSave document: Document, error: Error?) {
+        if pdfAnnotationsMutatedMoveToDocsDirectory, let filePathComponent = filePathComponent {
+            let to = URL.documentsDirectory.appendingPathComponent(filePathComponent)
+            if !FileManager.default.fileExists(atPath: to.path), let from = document.fileURL {
+                do {
+                    try FileManager.default.createDirectory(at: to.deletingLastPathComponent(), withIntermediateDirectories: true)
+                    if FileManager.default.fileExists(atPath: to.path) {
+                        try FileManager.default.removeItem(at: to)
+                    }
+                    try FileManager.default.moveItem(at: from, to: to)
```

Comment:
Interesting that the PSPDFKit Document doesn't mind this file being moved out from under it. Seems to work just fine though.